### ISSUE TITLE
Error registering custom alloy forge controller when custom alloy forge uses modded material as its base

### ISSUE THREAD

**Issue by truesephah** [LordDeatHunter/Alloy-Forgery#31]
Recently i've been trying to make a sort of hardcore style modpack. Alloy Forgery is perfect for this because of its customization. Im trying to gate landmark alloys behind individual alloy forge tiers, just to make progression a little bit more interesting. To do this, I added 3 individual forge bricks with KubeJS, instead of using vanilla blocks. 
The mod really doesnt like when I do this. It causes fabric to throw an error saying the crash was caused by whatever mod the configured alloy forge's base material happens to be from. KubeJS blocks throw: `Could not execute entrypoint stage 'main' due to errors, provided by 'kubejs'!` Just as a troubleshooting option I tried using polished tuff from Twigs, which threw: `Could not execute entrypoint stage 'main' due to errors, provided by 'twigs'!'`, and so on. In the ~10 mods i've tested, they've all behaved the same way when their blocks are used as alloy forge materials.
Full crash report can be found [here](https://pastebin.com/raw/C9Aa2zNs)

**Comment by truesephah** [comment]
json for custom forge, located in moddata/alloy_forgery/alloy_forges
`{
  "material": "kubejs:deepslate_forge_brick",
  "tier": 1,
  "speed_multiplier": 1,
  "fuel_capacity": 55000
}`


and here's the kubejs:
```
onEvent('block.registry', event => {
    event.create('deepslate_forge_brick', block => {
        block.material('stone')
        block.hardness(5)
        block.displayName('Deepslate Forge Brick')
    })
})

**Comment by Noaaan** [comment]
Thank you for reporting. Should be fixed as of owo-lib 0.3.14
